lib/main_common.pm

```diff
@@ -1591,7 +1591,7 @@ sub load_extra_tests_console {
     loadtest 'console/vhostmd';
     loadtest 'console/rpcbind' unless is_jeos;
     # sysauth test scenarios run in the console
-    loadtest "sysauth/sssd" if get_var('SYSAUTHTEST') || is_sle('12-SP5+');
+    loadtest "sysauth/sssd" if get_var('SYSAUTHTEST') || is_sle('15+');
```

Comment:
better make it:
```
if (get_var('SYSAUTHTEST')){
 loadtest "sysauth/sssd" if is_sle('12-SP5+')  && !check_var('BACKEND', 'svirt');
}

Reply:
updated, thanks!

Reply:
@Zaoliang Check my suggestion again, is_sle('12-SP5+')  && !check_var('BACKEND', 'svirt') is part of one condition and  get_var('SYSAUTHTEST')  is another

Reply:
yes, corrected now, thanks